modules/nodes-pods-vertical-autoscaler-tuning.adoc

```diff
@@ -20,7 +20,165 @@ You can perform the following tunings on the VPA components by editing the `Vert
 
 * To configure the VPA Operator to monitor only workloads that are being managed by a VPA CR, set the `memory-saver` parameter to `true` for the recommender component.
 
-The following example VPA controller CR sets the VPA API QPS and burts rates, configures the component pod resource requests, and sets `memory-saver` to `true` for the recommender:
+For guidelines on the resources and rate limits that you could set for each VPA component, the following tables provide baseline values, depending on the size of your cluster and other factors. 
+
+[IMPORTANT]
+====
+These values were derived by internal Red{nbsp}Hat testing on clusters that are not necessarily representative of the loads a real-world cluster would experience. You should test these values in a non-production cluster before configuring a production cluster. 
+====
+
+.Requests by containers in the cluster
+[cols="1,1,1,1,1,1,1,1,1,3,3"]
+|===
+| Component 2+| 1-500 containers 2+| 500-1000 containers 2+| 1000-2000 containers 2+| 2000-4000 containers 2+| 4000+ containers
+
+| 
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+| *CPU*
+| *Memory*
+
+s| Admission 
+| 25m 
+| 50Mi
+| 25m 
+| 75Mi 
+| 40m 
+| 150Mi 
+| 75m 
+| 260Mi 
+| (0.03 x c)/2 + 10 ^[1]^
```

Comment:
I think you can get rid of the `x`, because it might get confused with another variable.
So it would be: 
`(0.03c)/2 + 10`